toplevel/coqloop.ml

```diff
@@ -469,6 +469,16 @@ let rec vernac_loop ~state =
   (* Exception printing should be done by the feedback listener,
      however this is not yet ready so we rely on the exception for
      now. *)
+  | Sys_blocked_io ->
+    (* the parser doesn't like nonblocking mode, cf #10918 *)
+    let msg =
+      Pp.(strbrk "Coqtop needs the standard input to be in blocking mode." ++ spc()
+          ++ str "One way of clearing the non-blocking flag is through python:" ++ fnl()
```

Comment:
```suggestion
          ++ str "One way of clearing the non-blocking flag is through Python:" ++ fnl()
```
(Assuming we're talking about the language and not the binary)

Reply:
Not worth a rebase, will add this change to #11457